stencil-library/src/components/business-form/business-form-stepped.stories.tsx

```diff
@@ -1,7 +1,15 @@
-interface BusinessFormSteppedStoryArgs {
-  authToken: string;
-  businessId: string;
-  cssVariables: string;
+class BusinessFormSteppedStoryArgs {
+  'auth-token': string;
+  'account-id': string;
+  'bussiness-id': string;
+  'cssVariables': string;
```

Comment:
Minor thing, but this could also be `css-variables` (kebab-case), just to not diverge with the others.

Reply:
Got that updated too - thanks Tiago.